src/ppx_deriving_yojson.ml

```diff
@@ -206,7 +206,7 @@ and desu_expr_of_typ ~path typ =
                  deriver (Ppx_deriving.string_of_core_type typ)
 
 let wrap_runtime decls =
-  [%expr let open Ppx_deriving_yojson_runtime in [%e decls]]
+  [%expr (let open Ppx_deriving_yojson_runtime in [%e decls]) [@ocaml.warning "-33"]]
```

Comment:
Thanks, fixed in https://github.com/whitequark/ppx_deriving_yojson/commit/cc29f3572d992a711036c42ab595f2c1e669e846